src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -718,9 +718,9 @@ describe('ReactDOMComponent', function() {
       expect(function() {
         React.render(<div style={1}></div>, container);
       }).toThrow(
-        'Invariant Violation: The `style` prop expects a mapping from style ' +
-        'properties to values, not a string. For example, ' +
-        'style={{marginRight: spacing + \'em\'}} when using JSX.'
+        'Invariant Violation: A dom node has incorrect inline style syntax.' +
+        ' The `style` prop expects a mapping from style properties to values, not' +
+        ' a string. For example, style={{marginRight: spacing + \'em\'}} when using JSX. '
```

Comment:
Please add a new test confirming that the addendum is actually added when you expect it to be.